src/styles/_map.scss

```diff
@@ -19,3 +19,9 @@
     display: none;
   }
 }
+
+@media (min-width: 641px) {
```

Comment:
I think you could just use `max-width` here?

We should create an SCSS variable for the moble breakpoint in `variables.scss`
Then anywhere it's used we can do things like `@media (max-width: #{$mobileBreakpoint}) {` note the wrapping of the variable in `#{ }`.

I should have done this when porting the CSS to SCSS but missed it. If you wouldn't mind correcting that please go for it!

Reply:
Got rid of that part so it's not necessary